content/en/aws/docdb/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "DocumentDB (DocDB)"
+linkTitle: "DocumentDB (DocDB)"
+categories: ["LocalStack Pro"]
+description: >
+  DocumentDB (DocDB)
+---
+
+LocalStack supports a basic version of [Amazon DocumentDB](https://aws.amazon.com/documentdb/) for testing.
+
+When loading the DocumentDB service, LocalStack will create a MongoDB container to handle DocumentDB storage. By default, this instance will listen on a port in the 33000-34000 port range and will instantiate a master user named 'test' without a password. The `mongosh` client may be used to log in to and interact with the instance.
+
+{{< alert title="Note" >}}
+Because LocalStack utilizes a MongoDB container to provide DocumentDB storage, LocalStack may not have feature parity with Amazon DocumentDB. The database engine may support additional features that Amazon DocumentDB does not and vice versa.
+{{< /alert >}}
+
+## Example
```

Comment:
Can you please re-name this as `Getting started` and provide the description and purpose of the commands you are running here?